test/integration/test_forward.rb

```diff
@@ -24,18 +24,6 @@
 class ForwardTestBase < NetSSHTest
   include IntegrationTestHelpers
 
-  # @yield [pid, port]
-  def start_sshd_7_or_later(port = '2200')
```

Comment:
Remove this method because it's already been implemented in `IntegrationTestHelpers` that is mixed in.